docker-compose/integ-dependencies.yml

```diff
@@ -120,7 +120,7 @@ services:
     depends_on:
       - cadence
     environment:
-      - CADENCE_CLI_ADDRESS=cadence:7933
+      - CADENCE_CLI_ADDRESS=cadence:7833
```

Comment:
changing this may be breaking the script which is for registering `default` domain